core/templates/services/audio-bar-status.service.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to notify about status of audio bar. This services
+ * notifies that whether the upper audio bar component is expanded
+ * or collapsed.
```

Comment:
Why this service is needed? I mean it seems like that the audio-bar directive should only handle this I'm not sure about the usage of this service. (This sill help anyone changing this service to know what it can effect currently it's not clear to me)

Reply:
Yeah, the audio-bar directive is still handling this, it's just that the tutor-card directive needed to know the status of the audio bar. See [this](https://github.com/oppia/oppia/pull/8519#discussion_r407135869) conversation here.